### ISSUE TITLE
Support for "Emoji picker" keyboard shortcut action

### ISSUE THREAD

**Issue by lorenzofanchi** [rxhanson/Multitouch-Bugs#81]
Hi Ryan, it would be great to be able to set a keyboard action that spawns the emoji picker. I tried to map "Ctrl + Cmd + Space" to a key, but it does not seem to invoke the picker window when pressed.

Cheers

**Comment by rxhanson** [comment]
I looked into this a little bit, and if I were to implement it as a "macOS keys & shortcuts" action, I would be adding that identical shortcut. The problem with that keyboard shortcut is that it is an application specific menu item, and not all applications are guaranteed to have that menu item (Multitouch itself doesn't have it). Also, in my experience, the character viewer can be a little weird in that it sometimes just doesn't appear when I think it should, even upon the keyboard shortcut invoking the menu item. For most of my testing, ctrl+cmd+space worked.

I think it should be possible to remove that inconsistency with the keyboard shortcut by executing an AppleScript, similar to https://github.com/rxhanson/Multitouch-Bugs/issues/78. Hitting the menu item in the menu bar seems to remove that behavior with the keyboard shortcut in my limited testing.

If I get a chance to write an applescript for it, I'll post back here.

**Comment by lorenzofanchi** [comment]
> The problem with that keyboard shortcut is that it is an application specific menu item, and not all applications are guaranteed to have that menu item (Multitouch itself doesn't have it).

Good point, let me check if maybe I just wasn't in the right app for it to be triggered.
